### ISSUE TITLE
Project is dead

### ISSUE THREAD

**Issue by bestouff** [damko/freedommarks-browser-webextension#23]
Hi,

It looks like the project has been unmaintained since more than one year. I didn't found a viable alternative, and nowadays Freedommarks doesn't work anymore for me (I can read the bookmarks but I can't add any).  
How do you guys do ?

**Comment by sjbat** [comment]
You're right. It still works in Firefox, but chromium-based browsers refuse to cooperate. I don't know the alternative.

**Comment by damko** [comment]
hey guys, no the project is not dead but definitely not maintained because I've no time to focus on this. I'm aware there are PR waiting (and most likely they are good) but I need to inspect them a little before accepting them.

I do use my own plugin everyday and it works on Firefox. On Chrome/Chromium is broken since a while although it should work. Not sure what's wrong at this time.

I'll try to give some time to PR asap. Not sure what to say about future development.

**Comment by bestouff** [comment]
Which version of FF are you using ? Here under FF 68 my bookmarks are read-only.

**Comment by damko** [comment]
60.9.0esr

**Comment by saywebsolutions** [comment]
@damko do you have any plans to find additional maintainers? Or maybe try something like https://www.codeshelter.co/? Thanks, I'd like to use the extension and appreciate the work you've done on it

**Comment by lixuuu** [comment]
I'm a big fan of Firefox, one of the few alternative browsers to Google, including the rendering engine. 
I just looked on the Firefox store, it seems that you removed your extension. I can't find it with the keyword FreedomMarks
